flaskr/controller/calculate_controller.py

```diff
@@ -15,10 +12,19 @@ def calculate():
     numberB = content["numberB"]
     operation = content["operation"]
 
-    error = None
-    if not numberA:
-        error = "Number A is required"
-    elif not numberB:
-        error = "Number B is required"
+    print(type(numberA))
+    print(type(numberB))
+
+    if not isinstance(numberA, int) and not isinstance(numberA, float):
```

Comment:
gyanus, hogy ettol hatekonyabb validacio is van a djangoban.

Reply:
Nem találtam djangoban más típus validációt.